src/Kros.AspNetCore/Kros.AspNetCore.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <Version>2.46.2</Version>
+    <Version>2.46.3</Version>
```

Comment:
Zmenil by som druhé čísielko. Nejedná sa o chybu, ale o zmenu implementácie. (DOkonca podľa bežných pravidiel je by sa malo meniť prvé číslo. Ale to netreba)